site/src/api/api.ts

```diff
@@ -222,9 +222,21 @@ export const getTemplate = async (
 
 export const getTemplates = async (
   organizationId: string,
+  deprecated?: boolean,
```

Comment:
Would be better to have this as a `options` or `params` argument with the type `GetTemplatesOptions` or `GetTemplatesParams` as we have for other endpoints so you could pass that as a `param` directly.

Reply:
so to add a type like:

```typescript
export interface TemplateOptions {
  readonly deprecated?: boolean;
}
```